packages/types/src/discord.ts

```diff
@@ -2843,6 +2846,8 @@ export interface DiscordGuildMemberUpdate {
   pending?: boolean
   /** when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again, null or a time in the past if the user is not timed out. Will throw a 403 error if the user has the ADMINISTRATOR permission or is the owner of the guild */
   communication_disabled_until?: string
+  /** Guild member flags */
+  flags?: MemberFlag
```

Comment:
```suggestion
  flags?: string
```

Reply:
i need to check but for this you might be right based on what discord uses in the response.